src/components/Header/Header.jsx

```diff
@@ -1,20 +1,60 @@
+import { Link, animateScroll as scroll } from "react-scroll";
+
 const Header = () => {
   return (
     <header className="my-4">
       <nav className="py-4 px-6 border-2 rounded-full">
         <ul className="flex md:gap-10 gap-4 md:justify-start justify-between">
-          <a className="hover:text-bright-sun" href="#">
+          <Link
```

Comment:
With all the repeated code here, I would encourage you to move this to a local component to reduce the work needed to update it.

Also, why does your `offset` prop use many different values? You have -70, -200, -100 & -250, is there maybe a way to set these dynamically or add a comment/some way for you to know what the values correlate to?

Reply:
I created a local NavLink component, thank you for pointing this out.

As for offset prop values - I wanted to make sure they are aligned in the viewport and look nice :')